### PR TITLE
*: update TiDB parameters, delete command option file

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

- Update wording and format in TiDB cmd-line arguments
- Apply the alphabetic order for parameters in `configuration.md` to make it easier to locate
- Delete the obsolete command option file `reference/configuration/tidb-server/server-command-option.md`
- Add more omitted parameters to `tidb-server/configuration-file.md`

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1925/

### Which version does your change affect? 

dev, v3.0, v2.1

### CODE REVIEW

File: dev/reference/configuration/tidb-server/configuration.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-# Configuration Flags
+# Configuration Options
 
-TiDB is configurable using command-line flags and environment variables. The default TiDB ports are 4000 for client requests and 10080 for status report.
+When you start the TiDB cluster, you can use command-line options or environment variables to configure it. This document introduces TiDB's command options. The default TiDB ports are `4000` for client requests and `10080` for status report.
 
 ## `--advertise-address`
 
-- The IP address on which to advertise the apiserver to the TiDB server
+- The IP address through which to log into the TiDB server
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration.md]:
```suggestion
- The IP address through which the TiDB server is logged into
```

File: dev/reference/configuration/tidb-server/configuration.md [hunk 2]
```diff
@@ -125,9 +121,14 @@ TiDB is configurable using command-line flags and environment variables. The def
 
 - The status report port for TiDB server
 - Default: "10080"
-- This is used to get server internal data. The data includes [Prometheus metrics](https://prometheus.io/) and [pprof](https://golang.org/pkg/net/http/pprof/).
-- Prometheus metrics can be got through `"http://host:status_port/metrics"`.
-- Pprof data can be got through `"http://host:status_port/debug/pprof"`.
+- This port is used to get server internal data. The data includes [Prometheus metrics](https://prometheus.io/) and [pprof](https://golang.org/pkg/net/http/pprof/).
+- Prometheus metrics can be accessed by `"http://host:status_port/metrics"`.
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration.md]:
```suggestion
- Prometheus metrics can be accessed by <http://host:status_port/metrics>.
```

File: dev/reference/configuration/tidb-server/configuration.md [hunk 3]
```diff
@@ -125,9 +121,14 @@ TiDB is configurable using command-line flags and environment variables. The def
...
+- pprof data can be accessed by `"http://host:status_port/debug/pprof"`.
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration.md]:
```suggestion
- pprof data can be accessed by <http://host:status_port/debug/pprof>.
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 4]
```diff
@@ -4,79 +4,101 @@ summary: Learn the TiDB configuration file options that are not involved in comm
...
 ---
 
-# TiDB Configuration File Description
+<!-- markdownlint-disable MD001 -->
 
-The TiDB configuration file supports more options than command line options. You can find the default configuration file in [config/config.toml.example](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) and rename it to `config.toml`.
+# TiDB Configuration File Description
 
-This document describes the options that are not involved in command line options. For command line options, see [Configuration Flags](/v2.1/reference/configuration/tidb-server/configuration.md).
+The TiDB configuration file supports more options than command line options. You can find the default configuration file in [config/config.toml.example](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) and rename it to `config.toml`. This document describes only the options that are not involved in [command line options](/v2.1/reference/configuration/tidb-server/configuration.md).
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
The TiDB configuration file supports more options than command line options. You can download the default configuration file [config/config.toml.example](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) and rename it to `config.toml`. This document describes only the options that are not involved in [command line options](/v2.1/reference/configuration/tidb-server/configuration.md).
```

File: dev/reference/configuration/tidb-server/configuration.md (hunk 2 above)
Comment (lilin90) [reply dev/reference/configuration/tidb-server/configuration.md]: @yikeke It's written as **"http://host:status_port/metrics"** in the config file. So let's keep the quotation marks.

File: dev/reference/configuration/tidb-server/configuration.md (hunk 1 above)
Comment (lilin90) [reply dev/reference/configuration/tidb-server/configuration.md]: Ha...Different perspective

Comment (yikeke) [reply dev/reference/configuration/tidb-server/configuration.md]: through which 后面应该接定语从句？to do sth. 是短语，并不是个句子...

File: dev/reference/configuration/tidb-server/configuration.md (hunk 2 above)
Comment (yikeke) [reply dev/reference/configuration/tidb-server/configuration.md]: Okay.

**Pull request merged**